generators/docker-compose/index.js

```diff
@@ -315,7 +318,7 @@ module.exports = DockerComposeGenerator.extend({
             if(this.serviceDiscoveryType === 'consul'){
                 this.template('_consul.yml', 'consul.yml');
             }
-            if(this.serviceDiscoveryType !== 'no'){
+            if(this.serviceDiscoveryType !== false){
```

Comment:
Can we simplify this to `if(this.serviceDiscoveryType) ` since it could only be false or a value